packages/modules/viewer-react/src/types.ts

```diff
@@ -16,19 +16,23 @@ import type {
   RpcInterfaceDefinition,
 } from "@itwin/core-common";
 import type {
+  BlankConnectionProps,
   BuiltInExtensionLoaderProps,
-  FrontendHubAccess,
+  IModelAppOptions,
   IModelConnection,
-  MapLayerOptions,
   ScreenViewport,
-  TileAdmin,
-  ToolAdmin,
   ViewChangeOptions,
   ViewCreator3dOptions,
   ViewState,
 } from "@itwin/core-frontend";
 import type { Vector3d, XAndY, XYAndZ } from "@itwin/core-geometry";
 
+export type Without<T1, T2> = { [P in Exclude<keyof T1, keyof T2>]?: never };
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
What is the linter's issue with this? Is something here poor practice?

Reply:
Record worked fine, i dont love using `unknown` but eh. resolving this comment